tests/performance/specs/shopper/checkout.js

```diff
@@ -0,0 +1,355 @@
+        "accept-encoding": "gzip, deflate, br",
+        "accept-language": "en-US,en;q=0.9",
+        connection: "keep-alive",
+        host: `${base_host}`,
+        "sec-fetch-dest": "document",
+        "sec-fetch-mode": "navigate",
+        "sec-fetch-site": "same-origin",
+        "sec-fetch-user": "?1",
+        "upgrade-insecure-requests": "1",
+        "sec-ch-ua":
+          '"Chromium";v="88", "Google Chrome";v="88", ";Not A Brand";v="99"',
+        "sec-ch-ua-mobile": "?0",
+      },
+    });
+    homePageTrend.add(response.timings.duration);
+  });
+
+  sleep(randomIntBetween(`${think_time_min}`, `${think_time_max}`));
+
+  group("Shop Page", function () {
+    response = http.get(`${base_url}/shop`, {
+      headers: {
+        accept:
+          "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*//*;q=0.8,application/signed-exchange;v=b3;q=0.9",
+        "accept-encoding": "gzip, deflate, br",
+        "accept-language": "en-US,en;q=0.9",
+        connection: "keep-alive",
+        host: `${base_host}`,
+        "sec-fetch-dest": "document",
+        "sec-fetch-mode": "navigate",
+        "sec-fetch-site": "same-origin",
+        "sec-fetch-user": "?1",
+        "upgrade-insecure-requests": "1",
+        "sec-ch-ua":
+          '"Chromium";v="88", "Google Chrome";v="88", ";Not A Brand";v="99"',
+        "sec-ch-ua-mobile": "?0",
+      },
+    });
+    shopPageTrend.add(response.timings.duration);
+  });
+
+  sleep(randomIntBetween(`${think_time_min}`, `${think_time_max}`));
+
+  group("Product Page", function () {
+    response = http.get(`${base_url}/product/${product_url}`, {
+      headers: {
+        accept:
+          "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*//*;q=0.8,application/signed-exchange;v=b3;q=0.9",
+        "accept-encoding": "gzip, deflate, br",
+        "accept-language": "en-US,en;q=0.9",
+        connection: "keep-alive",
+        host: `${base_host}`,
+        "sec-fetch-dest": "document",
+        "sec-fetch-mode": "navigate",
+        "sec-fetch-site": "same-origin",
+        "sec-fetch-user": "?1",
+        "upgrade-insecure-requests": "1",
+        "sec-ch-ua":
+          '"Chromium";v="88", "Google Chrome";v="88", ";Not A Brand";v="99"',
+        "sec-ch-ua-mobile": "?0",
+      },
+    });
+    productPageTrend.add(response.timings.duration);
+  });
+
+  sleep(randomIntBetween(`${think_time_min}`, `${think_time_max}`));
+
+  group("Product Page Add to cart", function () {
+    response = http.post(
+      `${base_url}/?wc-ajax=add_to_cart`,
+      {
+        product_sku: `${product_sku}`,
+        product_id: `${product_id}`,
+        quantity: "1",
+      },
+      {
+        headers: {
+          accept: "application/json, text/javascript, */*; q=0.01",
+          "accept-encoding": "gzip, deflate, br",
+          "accept-language": "en-US,en;q=0.9",
+          connection: "keep-alive",
+          "content-type":
```

Comment:
for all post requests, removing this `content-type` header worked better, especially with atomic site. When this header was present, I was always getting an error. Let's remove for all post requests.

Reply:
moved headers to separate file to make it easier to control and tweak them.